Makefile

```diff
@@ -106,9 +106,6 @@ docker-login:
 
 deploy-ci: docker-login image push tag-latest push-latest
 
-update-opa:
```

Comment:
This target is currently used by the auto-revendor service. Do we plan to update that ?

Reply:
That revendor service would be removed when we switch to using this script and dependabot.

Reply:
Great !